examples/ExampleImgPickNG/app/package.json

```diff
@@ -0,0 +1,49 @@
+{
+    "private": false,
+    "nativescript": {
+        "id": "org.nativescript.helloworldng"
+    },
+    "name": "tns-template-hello-world-ng",
+    "main": "main.js",
+    "version": "2.3.4",
+    "author": "Telerik <support@telerik.com>",
+    "description": "Nativescript Angular Hello World template",
+    "license": "BSD",
+    "keywords": [
+        "telerik",
+        "mobile",
+        "angular",
+        "nativescript",
+        "{N}",
+        "tns",
+        "appbuilder",
+        "template"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "git://github.com/NativeScript/template-hello-world-ng"
+    },
+    "homepage": "https://github.com/NativeScript/template-hello-world-ng",
+    "android": {
+        "v8Flags": "--expose_gc"
+    },
+    "dependencies": {
```

Comment:
These dependencies are never installed. The package.json inside the app folder is packed in {N} app and provides some options for the app but dependencies should be added to the root package.json.